### ISSUE TITLE
Accessibility (and couple other minor things): Collection details page

### ISSUE THREAD

**Issue by astridu** [sul-dlss/hungry-hungry-hippo#600]
1. The Related links row is throwing an accessibility error because it thinks it's missing a cell for the table header. Maybe if it's one row with 2 cells it won't throw the error? (looking at a row up above that doesn't have a value)
2. Change 'Collection information', ' Details', and 'Links to related information' to H3s.
3. Add a little more white space between the breadcrumbs and the collection details tabs. In my mockups I have about 50 pixels between them.
